modules/porous_flow/test/tests/jacobian/eff_stress03.i

```diff
@@ -0,0 +1,119 @@
+# 2phase (PP)
+# vanGenuchten, constant-bulk density for each phase, constant porosity, 2components (that exist in both phases)
+# unsaturated
+# RZ coordinates
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 1
+  ny = 1
+[]
+
+[Problem]
+  coord_type = RZ
+[]
+
+[GlobalParams]
+  PorousFlowDictator = dictator
+[]
+
+[Variables]
+  [./ppwater]
+  [../]
+  [./ppgas]
+  [../]
+[]
+
+[AuxVariables]
+  [./massfrac_ph0_sp0]
+  [../]
+  [./massfrac_ph1_sp0]
+  [../]
+[]
+
+[ICs]
+  [./ppwater]
+    type = RandomIC
+    variable = ppwater
+    min = -1
+    max = 0
+  [../]
+  [./ppgas]
+    type = RandomIC
+    variable = ppgas
+    min = 0
+    max = 1
+  [../]
+  [./massfrac_ph0_sp0]
+    type = RandomIC
```

Comment:
If you are going to use RandomIC, you should restrict this test to run on one proc/one thread. We are eventually going to run the test suite in parallel in the modules (or we'd like to and it'll take a long time to clean these up). RandomIC doesn't generate the same random field in parallel. That's going to take more work.

Reply:
Aha, that's good to know about RandomIC and parallel runs.  However, in this case, does it matter?  This is just a PetscJacobian test - so any random numbers should produce a "zero" Jacobian and i shouldn't need the same random numbers each time.  Right?

Reply:
Good question, you're right. It probably doesn't matter in this case.

Reply:
Cool, thanks.  I found some other PorousFlow tests where we should restrict to serial, so ill include the changes in this PR.